cityofaustin/atd-vz-data
# Make the Dashboard page compatible with the new data model

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/18328

The issue suggests using the crashes list view but the way this currently works is we are supposed to reflect the same values in this VZE that are in the socrata dataset, so i thought it made the most sense to use the `socrata_export_crashes_view`

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Check out this branch, start up your data model db and apply hasura migrations/metadata and run the cris import script if necessary
2. Open up the hasura console and navigate to the `socrata_export_crashes_view`, sort by death count, years of life lost, and suspected serious injuries to see what we would expect the widgets on the dashboard to say. Keep in mind this is only for crashes that happened in 2024.
3. Start your VZE, go to the dashboard, and confirm you are seeing the numbers you would expect to see in the widgets.
4. Now go create some temp crash records with fatalities/serious injuries to further test the dashboard widget. Remember that the socrata dataset is only counting crashes that happened at least 14 days ago. So in your crash creation the date needs to be todays date - 14 days or farther back.
5. Also, you will need to update the private drive flag to `NO` in the crash details page of the temp record in order for it to factor into the socrata ytd totals. **_Sidenote - lets not let this requirement slip through the cracks so the VZ team isnt wondering why their temp crashes arent making it into the socrata totals_**
6. Update your computer to a different time zone. You should still see the same numbers in your widgets bc everything should be using UTC time and not expecting the user to be in CT.


---
#### Ship list
